README.md

```diff
@@ -4,79 +4,136 @@
 #    - Edit the title to match your project
 #    - Edit the description to describe your project
 
-# 3. Edit package.json
+# 5. Edit package.json
 #    - Update the title to match the title of this README
 #    - Update the description to match the description of this README
 #    - Edit the semantic version to match your project requirements
-#    - Change the Git details including repository, bugs and homepage
-
-# 4. Delete any unwanted files in /docker (or the entire /docker folder if you're not containerising)
 
-# 5. Delete the hidden .git folder created when you cloned
-rm -rf .git
+# 6. Delete any unwanted files in /docker (or the entire /docker folder if you're not containerising)
 
-# 6. Re-initialise .git
+# 7. Re-initialise .git
 git init
 
-# 7. Add your own repo's origin
-git remote add origin my-git-origin
+# 8. Add your own repo's origin
+git remote add origin {my-git-origin}
 
-# 8. Delete this instruction block
+# 9. Update package.json to reflect the new origin and clean up any other properties by following the prompts
+npm init
 
-# 9. Stage and commit files
+# 10. Delete this instruction block
+
+# 11. Stage and commit files
 git add -A
 git commit -m"Initial creation"
 
-# 10. Push
+# 12. Push
 git push -u origin master
+
+# 13. Create a develop branch (workflow dependent)
+git checkout -b develop
+git push -u origin develop
+
+# Your project and repo are ready for your first feature branch
+git checkout -b {my-feature-branch-name}
 ```
 
-## Development setup
+## For Project Developers
+
+Follow this section if you will be developing new features and fixing bugs for this repository.
 
-Development setup is very easy. Just clone this repository and start editing.
+### Prerequisites
+
+- A good code editor. We suggest [VSCode](https://code.visualstudio.com/), [Atom](https://atom.io/) or [SublimeText](https://www.sublimetext.com/).
+- [NodeJS LTS+ and NPM 6.14+](https://nodejs.org/en/download/)
+- [Git 2.25+](https://git-scm.com/downloads)
+
+### Setup and Configuration
 
 ```sh
-git clone git@github.com:tforster/git-new.git
-```
+# 1. Clone this repository as your new project
+git clone git@github.com:tforster/git-new.git /my/path/to/my/project-folder/
+cd /my/path/to/my/project-folder/git-new
 
-## Release History
+# 3. Create a new branch
+git checkout -b {new-branch-name}
+
+# 2. Make changes by editing files and/or using NPM to update dependent packages as well as package.json
+
+# 3. Stage changes
+git add -A
+
+# 4. Commit changes
+git commit -t"{detailed-commit-message}"
+
+# 5. Push changes
+git push origin {new-branch-name}
+
+# 6. Create a pull request
+```
 
-- 0.1.0 **Initial Creation** (2020-04-09)
+## Change log
 
-Initial creation of this project and Git repository.
+See [CHANGELOG.md](changelog.md)
 
 ## Meta
 
 Troy Forster – [@tforster](https://twitter.com/tforster) – troy.forster@gmail.com
 
 Distributed under the ISC license. See `LICENSE` for more information.
```

Comment:
Does this LICENSE file exist yet?

Reply:
It does not. It's probably a good habit to get into, adding an actual license file along with the simple mention of a license in package.json. It is another "control" file in the root though. I'm ok with that because it serves a unique purpose. It's not quite the same has having a katrillion dot-files all vying for some linting config attention.

What say you to adding LICENSE to our template. We could have two, one is our typical permissive OSS license (actual choice tbd) and the other we would reserve for private client projects. LICENSE-OSS.md and LICENSE-CLOSED.md and the instructions say delete one, rename the other.

Reply:
That sounds good to me.